src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,147 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    public static class Node<T> {
```

Comment:
1. There is no need to expose inner class
2. Then place it after public methods
```suggestion
    private static class Node<T> {
```